Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java

```diff
@@ -2114,18 +2201,18 @@ private DataType getExpressionDefResultType(ExpressionDef expressionDef) {
             return expressionDef.getResultType();
         }
 
-        // If the current expression context is patient, a reference to a population context expression will indicate a full
+        // If the current expression context is specific, a reference to an unspecified context expression will indicate a full
         // evaluation of the population context expression, and the result type is the same.
-        if (inPatientContext()) {
+        if (inSpecificContext()) {
             return expressionDef.getResultType();
         }
 
-        // If the current expression context is population, a reference to a patient context expression will need to be
-        // performed for every patient in the population, so the result type is promoted to a list (if it is not already).
-        if (inPopulationContext()) {
+        // If the current expression context is unspecified, a reference to a specific context expression will need to be
```

Comment:
Only a comment, but probably should replace `unspecified` with `unfiltered`

Reply:
Agreed, corrected